### PR TITLE
ss-10: Add Line Chart

### PR DESCRIPTION
**Pull request by Gandrejko**
image

### CODE REVIEW

**Comment by nikita-remeslov** [comment]
Conflicts

File: frontend/src/bundles/common/components/charts/line-chart/line.type.ts
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/charts/line-chart/line.type.ts:1]: Please check project structure 
image

File: frontend/src/bundles/common/components/charts/line-chart/chart.module.css
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/charts/line-chart/chart.module.css:1]: styles.module.scss

File: frontend/src/bundles/common/components/charts/line-chart/line.type.ts
Comment (nikita-remeslov) [reply frontend/src/bundles/common/components/charts/line-chart/line.type.ts:1]: Also props type define in the component

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/charts/line-chart/chart.module.css
Comment (Gandrejko) [reply frontend/src/bundles/common/components/charts/line-chart/chart.module.css:1]: fixed

File: frontend/src/bundles/common/components/charts/line-chart/line.type.ts
Comment (Gandrejko) [reply frontend/src/bundles/common/components/charts/line-chart/line.type.ts:1]: fixed

**Comment by Gandrejko** [comment]
https://www.loom.com/share/22d8a6bda824492f94818184b5559619

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**